.travis.yml

```diff
@@ -17,6 +17,8 @@ gemfile:
 
 matrix:
   exclude:
+    - rvm: 2.2.3
+      gemfile: Gemfile.rails5
```

Comment:
This branch will need to be rebased on latest master to fix merge conflicts, it's possible that the issue is fixed since then. If not, the issue needs to be investigated. We cannot simply stop testing a supported Rails/Ruby combination.